spec/support/pdf_matchers.rb

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rspec/expectations"
+require "pdf-reader"
+
+RSpec::Matchers.define :match_pdf_content do |fixtures_path_to_expected_pdf|
```

Comment:
Was wary of getting into trying to check the contents of PDF's, but think this is an awesome solution ❤️ 🎉 !

Just means I will now need to find time to go back and use it in our other PDF stuff 😩 .